cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/co/cask/cdap/etl/common/AbstractTransformContext.java

```diff
@@ -42,7 +42,7 @@ public AbstractTransformContext(PluginContext pluginContext, Metrics metrics, Lo
     this.pluginContext = pluginContext;
     this.stageId = stageId;
```

Comment:
as mentioned above, lets use 'name' instead of 'id' to be consistent with our terminology.